templates/security/cves/about.html

```diff
@@ -0,0 +1,438 @@
+                </td>
+                <td colspan="2">
+                  The package is affected by the issue for the given release, but fixing has been deferred. We usually provide an explanation, either next to the status or in the notes.
+                </td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">
+                  <i class="p-icon--success"></i>Fixed
+                </td>
+                <td colspan="2">
+                  The package was affected by the issue for the given release, but a version fixed by the Ubuntu Security team is now available. We usually indicate the package version that contains the patch.
+                </td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">
+                  <i class="p-icon--error-grey"></i>Ignored
+                </td>
+                <td colspan="2">
+                  The package, while related to the CVE in some way for the given release, won’t be fixed. This could be because the package is no longer supported, the CVE has been withdrawn,
+                  or fixing the issue would break other functionality. We usually provide an explanation, either next to the status or in the notes.
+                </td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">Not in release</td>
+                <td colspan="2">The package is not part of the given release.</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+
+        <h2 class="question-heading" id="export-data">Exporting CVE data</h2>
+        <div class="p-section">
+          <p>You can export our CVE data in two structured formats:</p>
+
+          <h3>Ubuntu OVAL (recommended)</h3>
+          <div>
+            <p>
+              Structured, machine-readable XML dataset for all supported Ubuntu releases, based on the Open Vulnerability and Assessment Language (OVAL).
+              It can be used to evaluate and manage security risks related to any existing Ubuntu components.
+            </p>
+          </div>
+          <p>
+            <a href="/security/oval">Documentation for Ubuntu OVAL&nbsp;&rsaquo;</a>
+          </p>
+
+          <h3>Ubuntu Security API</h3>
+          <div class="p-notification--caution is-inline">
+            <div class="p-notification__content">
+              <p class="p-notification__message">
+                The Ubuntu Security API is still a work in progress and subject to change. Please don’t use it in production environments or automations.
+              </p>
+            </div>
+          </div>
+          <div>
+            <p>
+              Open JSON API that provides endpoints for our CVE database and the <a href="/security/notices">Ubuntu Security Notices (USN)</a>.
+              You can find links to the JSON endpoints in the search pages and the individual CVE pages.
```

Comment:
If indeed we're merging this one into main right away, this sentence won't be true. There are no links to any of the endpoints on the existing cve search page

Reply:
I'll change it so that it merges to the feature branch, makes more sense that way